source/_integrations/risco.markdown

```diff
@@ -90,9 +90,20 @@ And in the reverse direction:
 | Arm Away | Arm |
 | Arm Home | Partial Arm |
 
-## Supported platforms:
+## Supported platforms
 
 - [Alarm control panel](/integrations/alarm_control_panel/)
 - [Binary sensor](/integrations/binary_sensor/)
 - [Sensor](/integrations/sensor/)
 - [Switch](/integrations/switch/)
+
+## Actions
+
+### Set time
+
+This service enables you to set the time of a panel on a location connection.
```

Comment:
```suggestion
The `risco.set_time action` enables you to set the time of a panel on a location connection.
```
I would not use the term service as it's called action now.

Reply:
100% correct. I'm still using 'service' all over the place in my configuration so my brain hasn't transitioned to using 'action' yet.